### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shubham-shahh**
minor fixes

### CODE REVIEW

**Comment by shubham-shahh** [comment]
Hi, @cenit can we integrate this small fixes?
thanks

File: README.md [hunk 1]
```diff
@@ -645,7 +645,7 @@ Example of custom object detection: `darknet.exe detector test data/obj.data yol
 
 - my Loss is very high and mAP is very low, is training wrong? Run training with `-show_imgs` flag at the end of training command, do you see correct bounded boxes of objects (in windows or in files `aug_...jpg`)? If no - your training dataset is wrong.
 
-- for each object which you want to detect - there must be at least 1 similar object in the Training dataset with about the same: shape, side of object, relative size, angle of rotation, tilt, illumination. So desirable that your training dataset include images with objects at different: scales, rotations, lightings, from different sides, on different backgrounds - you should preferably have 2000 different images for each class or more, and you should train `2000*classes` iterations or more
```
Comment (cenit) [inline README.md:648]: i think side here was meant to be that, not size

**Review by cenit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shubham-shahh) [reply README.md:648]: okay, makes sense, I'll revert it. thanks

Comment (shubham-shahh) [reply README.md]: I reverted the changes,

Comment (shubham-shahh) [reply README.md]: Hi @cenit , I made the required changes. I was planning to add comments in the source code explaining a summary of the respective functions which will help users to understand the code better and make the required changes. @AlexeyAB What are your thoughts on this?
thanks

**Pull request closed without merging**